### ISSUE TITLE
PL incompatible?

### ISSUE THREAD

**Issue by Stebulous** [MyBBStuff/MyAlerts#219]
https://i.imgur.com/G3cgdti.png

excuse my stupidity but shouldn't plugin library work on 1.8.12?

**Comment by metonator** [comment]
PluginLibrary doesn't require installation. plugins using that library does

**Comment by euantorano** [comment]
Yeah, you should be fine because as said above PL doesn't need installed.

There is a newer 1.8 compatible version available here though: https://github.com/frostschutz/MyBB-PluginLibrary
